Disappearing dataset repos

These dataset versions cannot be found anymore: (from ci tests)
```
FAILED tests/test_all_abstasks.py::test_dataset_availability - AssertionError: Datasets not available on Hugging Face:
  hard - revision b108d2c32ee4e1f4176ea233e1a5ac17bceb9ef9
  poem_sentiment - revision 329d529d875a00c47ec71954a1a96ae167584770
  hindi_discourse - revision 218ce687943a0da435d6d62751a4ab216be6cd40
  mlsum - revision b5d54f8f3b61ae17845046286940f03c6bc79bc7
  mlsum - revision b5d54f8f3b61ae17845046286940f03c6bc79bc7
  mlqa - revision 397ed406c1a7902140303e7faf60fff35b58d285
  told-br - revision fb4f11a5bc68b99891852d20f1ec074be6289768
```

--- a/mteb/tasks/Classification/ara/HotelReviewSentimentClassification.py
+++ b/mteb/tasks/Classification/ara/HotelReviewSentimentClassification.py
@@ -10,7 +10,7 @@ class HotelReviewSentimentClassification(AbsTaskClassification):
     metadata = TaskMetadata(
         name="HotelReviewSentimentClassification",
         dataset={
-            "path": "hard",
+            "path": "Elnagara/hard",
             "revision": "b108d2c32ee4e1f4176ea233e1a5ac17bceb9ef9",
         },
         description="HARD is a dataset of Arabic hotel reviews collected from the Booking.com website.",

--- a/mteb/tasks/Classification/eng/PoemSentimentClassification.py
+++ b/mteb/tasks/Classification/eng/PoemSentimentClassification.py
@@ -10,7 +10,7 @@ class PoemSentimentClassification(AbsTaskClassification):
         description="Poem Sentiment is a sentiment dataset of poem verses from Project Gutenberg.",
         reference="https://arxiv.org/abs/2011.02686",
         dataset={
-            "path": "poem_sentiment",
+            "path": "google-research-datasets/poem_sentiment",
             "revision": "329d529d875a00c47ec71954a1a96ae167584770",
         },
         type="Classification",

--- a/mteb/tasks/Classification/hin/HindiDiscourseClassification.py
+++ b/mteb/tasks/Classification/hin/HindiDiscourseClassification.py
@@ -8,7 +8,7 @@ class HindiDiscourseClassification(AbsTaskClassification):
     metadata = TaskMetadata(
         name="HindiDiscourseClassification",
         dataset={
-            "path": "hindi_discourse",
+            "path": "midas/hindi_discourse",
             "revision": "218ce687943a0da435d6d62751a4ab216be6cd40",
         },
         description="A Hindi Discourse dataset in Hindi with values for coherence.",

--- a/mteb/tasks/Clustering/multilingual/MLSUMClusteringP2P.py
+++ b/mteb/tasks/Clustering/multilingual/MLSUMClusteringP2P.py
@@ -21,7 +21,7 @@ class MLSUMClusteringP2P(AbsTaskClustering, MultilingualTask):
         description="Clustering of newspaper article contents and titles from MLSUM dataset. Clustering of 10 sets on the newpaper article topics.",
         reference="https://huggingface.co/datasets/mlsum",
         dataset={
-            "path": "mlsum",
+            "path": "reciTAL/mlsum",
             "revision": "b5d54f8f3b61ae17845046286940f03c6bc79bc7",
             "trust_remote_code": True,
         },

--- a/mteb/tasks/Clustering/multilingual/MLSUMClusteringS2S.py
+++ b/mteb/tasks/Clustering/multilingual/MLSUMClusteringS2S.py
@@ -21,7 +21,7 @@ class MLSUMClusteringS2S(AbsTaskClustering, MultilingualTask):
         description="Clustering of newspaper article contents and titles from MLSUM dataset. Clustering of 10 sets on the newpaper article topics.",
         reference="https://huggingface.co/datasets/mlsum",
         dataset={
-            "path": "mlsum",
+            "path": "reciTAL/mlsum",
             "revision": "b5d54f8f3b61ae17845046286940f03c6bc79bc7",
             "trust_remote_code": True,
         },

--- a/mteb/tasks/MultiLabelClassification/por/BrazilianToxicTweetsClassification.py
+++ b/mteb/tasks/MultiLabelClassification/por/BrazilianToxicTweetsClassification.py
@@ -15,7 +15,7 @@ class BrazilianToxicTweetsClassification(AbsTaskMultilabelClassification):
         """,
         reference="https://paperswithcode.com/dataset/told-br",
         dataset={
-            "path": "told-br",
+            "path": "JAugusto97/told-br",
             "revision": "fb4f11a5bc68b99891852d20f1ec074be6289768",
             "name": "multilabel",
         },

--- a/mteb/tasks/Retrieval/multilingual/MLQARetrieval.py
+++ b/mteb/tasks/Retrieval/multilingual/MLQARetrieval.py
@@ -84,7 +84,7 @@ class MLQARetrieval(AbsTaskRetrieval, CrosslingualTask):
         4 different languages on average.""",
         reference="https://huggingface.co/datasets/mlqa",
         dataset={
-            "path": "mlqa",
+            "path": "facebook/mlqa",
             "revision": "397ed406c1a7902140303e7faf60fff35b58d285",
         },
         type="Retrieval",

